### PR TITLE
[debian] replace stretch by buster

### PR DESCRIPTION
**Pull request by arthurzenika**

### CODE REVIEW

File: profiles/default.preseed [hunk 1]
```diff
@@ -69,7 +69,7 @@ d-i apt-setup/services-select multiselect security, updates
 d-i apt-setup/security_host string security.debian.org
 
 d-i apt-setup/local0/repository string \
-	http://forge.yunohost.org/debian/ stretch stable
+	http://forge.yunohost.org/debian/ buster stable
```
Comment (arthurzenika) [inline profiles/default.preseed:72]: Where is the builder for these packages ?

File: add-firmware-to [hunk 2]
```diff
@@ -99,7 +99,7 @@ if [ -e "$outfile" ]; then
 fi
 
 suite="$3"
-if [ "$suite" != "stretch" ] && [ "$suite" != "jessie" ]; then
+if [ "$suite" != "buster" ] && [ "$suite" != "stretch" ] && [ "$suite" != "jessie" ]; then
```
Comment (arthurzenika) [inline add-firmware-to]: We can probably drop jessie from this ?

**Comment by arthurzenika** [comment]
Related to https://github.com/YunoHost/issues/issues/1255

File: profiles/default.preseed (hunk 1 above)
Comment (alexAubin) [reply profiles/default.preseed:72]: That happens in https://github.com/YunoHost/vinaigrette (which is kind of a mess ... but at least it's more organized and versionned than 2 years ago :/ )

File: add-firmware-to (hunk 2 above)
Comment (arthurlutz) [inline add-firmware-to]:
```suggestion
if [ "$suite" != "buster" ] && [ "$suite" != "stretch" ]; then
```

**Review by kay0u — APPROVED** [review]
Tested on my side, works!

**Pull request merged**